src/odemis/dataio/csv.py

```diff
@@ -112,7 +154,10 @@ def export(filename, data):
             data = numpy.append(phi_lin, data, axis=0)
 
             # Set the 'header' in the 0,0 element
-            first_row = ['theta\phi[rad]'] + [d for d in data[0, 1:]]
+            first_row = ['theta\\phi[rad]'] + [d for d in data[0, 1:]]
             csv_writer.writerow(first_row)
             # dump the array
             csv_writer.writerows(data[1:, :])
+            
+    else:
+        raise IOError("Unknown acquisition type of data to be exported as CSV")
```

Comment:
In such case, a `ValueError` is more appropriate (as it's a sign that the function input is incorrect).